android-plugin/src/main/java/io/ecocode/java/Java.java

```diff
@@ -20,11 +20,11 @@
 package io.ecocode.java;
 
 public interface Java { //NOSONAR - we use the interface for constant without inheriting it
-    String REPOSITORY_NAME = "ecoCode";
-    String PROFILE_NAME = "ecoCode";
+    String REPOSITORY_NAME = "ecoCode (Android)";
+    String PROFILE_NAME = "ecoCode (Android)";
     String KEY = "java";
-    String REPOSITORY_KEY = "ecoCode-java";
+    String REPOSITORY_KEY = "ecocode-android-java";
     // don't change that because the path is hard coded in CheckVerifier
-    String JAVA_RESOURCE_PATH = "/org/sonar/l10n/java/rules/squid"; //NOSONAR - tis URI is the same everywhere
+    String JAVA_RESOURCE_PATH = "/io/ecocode/rules/java"; //NOSONAR - tis URI is the same everywhere
     String PROFILE_PATH = "org/sonar/l10n/java/rules/squid/ecocode_java_profile.json";
```

Comment:
why do we keep the old path ?

Reply:
I don't understand your question.
Indeed, I change the path of rules to the new one: io/ecocode/rules/java

Reply:
please see my answer above

Reply:
I think this (implementation-related) profile should not be the same as the specification path.
Thus, the profile must share the same path as the implementation and not the same specification path.
I will change this profile path based on this explanation.